src/ReactTouchHandler.js

```diff
@@ -148,6 +148,7 @@ class ReactTouchHandler {
   }
 
   onTouchMove(/*object*/ event) {
+    event.preventDefault();
```

Comment:
if the delta is zero, `preventDefault` wasn't being called. This is why the parent container / page get's scrolled
when the table is at any start/end scroll position

Reply:
Will this lead to the issue where you can't scroll the parent page with your pointer over the grid?

Reply:
@wcjordan , Yes exactly. My reasoning was that I wouldn't want to scroll any container if the cursor is on the table. But now I agree, that it's better to have a FF control this instead (https://github.com/schrodinger/fixed-data-table-2/issues/419#issuecomment-479531587). Will add